src/main/java/codesquad/springcafe/controller/CommentApiController.java

```diff
@@ -0,0 +1,54 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.form.comment.CommentWriteForm;
+import codesquad.springcafe.form.user.LoginUser;
+import codesquad.springcafe.model.Comment;
+import codesquad.springcafe.service.ArticleService;
+import codesquad.springcafe.service.CommentService;
+import jakarta.validation.Valid;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.SessionAttribute;
+
+@RestController
+@RequestMapping("/api/articles/detail/{articleId}/comments")
```

Comment:
```suggestion
@RequestMapping("/api/articles/{articleId}/comments")
```

요게 맞지 않을까요? 아닐 수도 있고요 ㅋ